li-utils/src/main/javaPegasus/com/linkedin/common/urn/JoinUrn.java

```diff
@@ -0,0 +1,66 @@
+package com.linkedin.common.urn;
+
+import com.linkedin.data.template.Custom;
+import com.linkedin.data.template.DirectCoercer;
+import com.linkedin.data.template.TemplateOutputCastException;
+import java.net.URISyntaxException;
+
+
+public class JoinUrn extends Urn {
```

Comment:
I'm a little confused, why is this class needed?

Reply:
Join will be a entity by itself.
This was proposed in the model:
https://docs.google.com/document/d/1JPtHbNEY-_E9ZwsCsFNPks35g1WVyWQEFd_K0fLF0wk/edit#heading=h.1t4gdnvyeais

Reply:
You can check this directory:
https://github.com/datahub-project/datahub/tree/master/li-utils/src/main/javaPegasus/com/linkedin/common/urn